Assets/Aptabase/Scripts/AptabaseSettings.cs

```diff
@@ -0,0 +1,12 @@
+using UnityEngine;
+
+namespace AptabaseSDK
```

Comment:
Not too strongly opinionated on this one, but I was thinking of using `Aptabase.Unity` to match the pattern we have as the other package is `Aptabase.Maui`. What you think?

Reply:
Oh, that's true. In that case maybe we can keep it like this, or check what other Unity SDKs are doing to keep it consistent.